### DISCUSSION TITLE
AppSwitcherContentView / Home Grabber / SBSwitcherWindow:Main

### DISCUSSION THREAD

**Discussion by sregg** [mobile-dev-inc/Maestro#1593] [Q&A] [answered]
In our React Native app, we see this full screen overlay layer that prevents Maestro from clicking on any element on the screen.

<img width="2032" alt="image" src="https://github.com/mobile-dev-inc/maestro/assets/1479283/9181e0f7-f741-4960-82a0-66ff8ba292be">

<img width="2032" alt="image" src="https://github.com/mobile-dev-inc/maestro/assets/1479283/f167e93c-0de6-4ac5-bff6-1d6f544ccf05">

Has anyone seen this before?

It doesn't seem to be there when building a release build instead of debug.

**Comment by galgord** [comment]
@sregg did you resolve this? We are having the same issue.

**Comment by john-trieu-nguyen** [comment, marked as answer]
I ran into this issue as well in React Native. 
Narrowed it down to having nested components on iOS.

```
<Pressable onPress={Keyboard.dismiss}>
    <View style={{flex: 1}}>
        {children}
    </View>
</Pressable>
```
The fix was adding accessible={false} to the "Pressable" parent component.
This should let you access the nested components in Maestro.

It is also mentioned in the [Maestro docs](https://maestro.mobile.dev/platform-support/react-native#interacting-with-nested-components-on-ios).

@galgord hope this helps!
